src/pages/RepoPage/CoverageTab/CoverageTab.jsx

```diff
@@ -25,13 +27,26 @@ const Loader = () => (
 )
 
 function CoverageTab() {
+  const { provider, owner, repo, branch } = useParams()
   const { data: repoData } = useRepoSettingsTeam()
   const { multipleTiers } = useFlags({
     multipleTiers: false,
   })
-  const { provider, owner } = useParams()
   const { data: tierName } = useTier({ provider, owner })
 
+  const { data } = useCoverageTabData({
+    provider,
+    owner,
+    repo,
+    branch: branch,
+  })
+
+  let displaySunburst = false
+  const fileCount = data?.branch?.head?.totals?.fileCount
+  if (typeof fileCount === 'number' && fileCount <= 200_000) {
```

Comment:
should this be 10_000? And maybe we can set to a const to avoid having magic numbers; like MAX_FILES_FOR_SUNBURST